server/src/routes/api/v1/ReviewsRouter.js

```diff
@@ -25,10 +25,16 @@ reviewsRouter.post('/', async (req, res) => {
 })
 
 reviewsRouter.delete("/:id", async (req,res)=>{
+  const { id } = req.user
   try {
-    await Review.query().findById(req.params.id).delete()
-
-    return res.status(204).json({ message: "review has been deleted!" })
+  const { userId } = await Review.query().findById(req.params.id) 
+    if(userId === id){
+      await Review.query().findById(req.params.id).delete()
+      return res.status(204).json({ message: "review has been deleted!" })
+    }
+    else {
+      return res.status(200)
```

Comment:
proper formatting for an else:
```js
if (something) {
  // block of code
} else {
  // block of code
}
```

this last else isn't doing anything - something needs to be returned with the response with `.json` - if we are checking to make sure that the user matches, the else would catch if they do not match, in which we would want to send back some message indicating that the action is not possible
```js
res.status(401).json({ message: "you are not authorized to delete this review" })
```